runtime/cere-dev/src/lib.rs

```diff
@@ -142,7 +146,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 54004,
+	spec_version: 54100,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
 	transaction_version: 18,
```

Comment:
You must increase the `transaction_version` as you changed existing extrinsics and added new ones